base/heartbeat.go

```diff
@@ -65,8 +65,9 @@ type HeartbeatListener interface {
 //       e.g  Default for a 4 node cluster: 12 ops/second
 type couchbaseHeartBeater struct {
 	bucket                  Bucket
-	nodeUUID                string
 	keyPrefix               string
+	groupID                 string
+	nodeUUID                string
```

Comment:
I'm not sure why nodeUUID is moving around here and below - better to avoid that kind of unnecessary diff.

Reply:
Moved it as thought it might be best to have it in the order that it is in the key. Will change back